### PR TITLE
Feature/learner dashboard

### PR DESCRIPTION
**Pull request by Ezeh20**
- This PR contains the home section of the learner's dashboard
- Routed the dashboard path
- Created a table component
- Made the layout for other sections of the dashboard
- Made room for the sidebar

**/dashboard navigates to the dashboard** 

**The main navbar has a fixed position which make  contents go behind it, it's either we change it to sticky and a top of 0
or we do a margin-top for every page** 

**The last screenshot is not included in the file**

Screenshot (44)
Screenshot (45)
Screenshot (46)

### CODE REVIEW

File: src/components/Table/internals/Table2/Table2.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                        const { id, Stipend_Category, Application_Status, Date_of_submission, Time_of_submission, Action } = itm;
+                        return (
+                            <tr key={id}>
+                                <td>{id}</td>
+                                <td>{Stipend_Category}</td>
+                                <td>
+                                    <span className={
+                                        Application_Status === 'Approved' ?
+                                            'approved bold_weight'
+                                            : Application_Status === 'In review' ? 'in_view bold_weight'
```
Comment (edustipend-root) [inline src/components/Table/internals/Table2/Table2.jsx:27]: Is there a way we can avoid having hard-coded strings?

File: src/components/Table/internals/Table2/Table2.module.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    margin: 0;
+    font-family: 'Manrope';
+    box-shadow: 0px 9px 38px 0px var(--box-shadow);
+}
+
+.table thead th {
+    text-align: left;
+    border-bottom: 1px solid rgba(0, 0, 0, 0.08);
+    font-weight: 600;
+    color: #2D3740;
```
Comment (edustipend-root) [inline src/components/Table/internals/Table2/Table2.module.css:15]: Can we add this color to a CSS variable?

File: src/components/Table/internals/Table2/Table2.module.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.head {
+    background-color: #ECEDEF;
```
Comment (edustipend-root) [inline src/components/Table/internals/Table2/Table2.module.css:19]: Add this as a CSS variable in `colors.css`

**Review by edustipend-root — CHANGES_REQUESTED** [review]
Looks good @Ezeh20, great job and thanks for adding the screenshots.
Please take note of the comments I made and when you address those, I'll get it merged.

**Comment by Ezeh20** [comment]
@edustipend-root I have made the changes

**Comment by Ezeh20** [comment]
> Looks good @Ezeh20, great job and thanks for adding the screenshots. Please take note of the comments I made and when you address those, I'll get it merged.

Done

**Review by edustipend-root — APPROVED** [review]

**Pull request merged**